src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -149,19 +150,10 @@ class JesBackend extends Backend with LazyLogging {
   /**
    * Delete authentication file in gcs once workflow is in a terminal state.
    */
-  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = {
-    Future(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
-  }
+  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = authenticated { connection =>
+    Future(connection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
+  } (workflow)
```

Comment:
So the whole point for this implicit stuff was to be able to pass the workflow descriptor to the authentication function, so it can extract the refreshToken from the option and use that in the GoogleCredentials on the fly.
But since we don't do any of that stuff anymore it removes the need for workflowDescriptor so I'll remove it.